docs/tutorial/vqe_for_hydrogen.ipynb

```diff
@@ -0,0 +1,397 @@
+      "text/plain": [
+       "<Figure size 1731.36x280 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from qamomile.core.ansatz.efficient_su2 import create_efficient_su2_circuit\n",
+    "from qamomile.core.circuit.drawer import plot_quantum_circuit\n",
+    "\n",
+    "ansatz = create_efficient_su2_circuit(\n",
+    "    hamiltonian.num_qubits, rotation_blocks=[\"ry\", \"rz\"],\n",
+    "    reps=4, entanglement=\"linear\"\n",
+    ")\n",
+    "plot_quantum_circuit(ansatz)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run VQE with Qiskit\n",
+    "\n",
+    "In this section, we execute the VQE using the Qiskit Aer simulator after converting the Qamomile Hamiltonian and ansatz to Qiskit format.  \n",
+    "Of course, you can use other quantum computing frameworks."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import qamomile.qiskit as qm_qk\n",
+    "\n",
+    "qk_transpiler = qm_qk.QiskitTranspiler()\n",
+    "qk_ansatz = qk_transpiler.transpile_circuit(ansatz)\n",
+    "qk_hamiltonian = qk_transpiler.transpile_hamiltonian(hamiltonian)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "         Current function value: -1.105933\n",
+      "         Iterations: 50\n",
+      "         Function evaluations: 2091\n",
+      "         Gradient evaluations: 51\n",
+      "  message: Maximum number of iterations has been exceeded.\n",
+      "  success: False\n",
+      "   status: 1\n",
+      "      fun: -1.1059333522848613\n",
+      "        x: [ 9.087e-01  1.723e+00 ...  2.929e+00  1.502e-01]\n",
+      "      nit: 50\n",
+      "      jac: [-1.237e-06  4.470e-08 ...  1.073e-06 -5.215e-07]\n",
+      " hess_inv: [[ 3.473e+00  1.455e+00 ... -9.225e-01  7.253e-01]\n",
+      "            [ 1.455e+00  2.812e+00 ... -4.512e-01  3.994e-01]\n",
+      "            ...\n",
+      "            [-9.225e-01 -4.512e-01 ...  2.418e+00  1.188e-01]\n",
+      "            [ 7.253e-01  3.994e-01 ...  1.188e-01  2.626e+00]]\n",
+      "     nfev: 2091\n",
+      "     njev: 51\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/yuyamashiro/Library/Caches/pypoetry/virtualenvs/qamomile-s0Pfpxir-py3.10/lib/python3.10/site-packages/scipy/optimize/_minimize.py:726: OptimizeWarning: Maximum number of iterations has been exceeded.\n",
+      "  res = _minimize_bfgs(fun, x0, args, jac, callback, **options)\n"
+     ]
+    }
+   ],
+   "source": [
+    "import qiskit.primitives as qk_pr\n",
+    "import qiskit as qk\n",
+    "from qiskit_aer import AerSimulator\n",
```

Comment:
`qiskit_aer` is independent package, so I think you can add cell to install this package at the top of this page.